### DISCUSSION TITLE
How to configure Renovate to update both versions if two versions are present in a Docker image tag?

### DISCUSSION THREAD

**Discussion by xuhdev** [renovatebot/renovate#30069] [Request Help]
### What would you like help with?

Other

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Consider the [Python Docker image](https://hub.docker.com/_/python). One may use:

```dockerfile
FROM python:3.12.4-alpine3.20
```

By default, Renovate only updates the `3.12.4` part to a later version once released. However, Renovate does not update to `3.12.4-alpine3.21` when it is released. How can I make Renovate update both versions?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by RahulGautamSingh** [comment]
We have a open feature request to support such cases: https://github.com/renovatebot/renovate/issues/27012
You can find some workarounds in the discussions linked in the above issue.
